archetypes/helidon/src/main/archetype/common/observability.xml

```diff
@@ -201,18 +223,28 @@ curl -H 'Accept: application/json' -X GET http://localhost:8080/metrics
                             <output>
                                 <model>
                                     <list key="dependencies">
-                                        <map if="${flavor} == 'mp'">
-                                            <value key="groupId">io.helidon.integrations.micrometer</value>
-                                            <value key="artifactId">helidon-integrations-micrometer-cdi</value>
+                                        <map>
+                                            <value key="groupId">io.helidon.metrics</value>
+                                            <value key="artifactId">helidon-metrics-api</value>
+                                            <value key="scope">compile</value>
```

Comment:
Similar question as above about the scoping for `helidon-metrics`. For SE apps that will use the neutral API it definitely makes sense to be compile-scoped. But for MP, runtime should (normally) be sufficient. That's assuming that the code uses the MP metrics API and not the Helidon neutral metrics API.